code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -175,7 +175,7 @@
 		/obj/item/stack/sheet/glass/cyborg = 50,
 		/obj/item/stack/sheet/rglass/cyborg = 50,
 		/obj/item/stack/cable_coil/cyborg = 50,
-		/obj/item/stack/rods = 30,
+		/obj/item/stack/rods/cyborg = 60,

```

Comment:
>it doesn't matter that much since enough time in a charger will bring a cyborg to a full stack anyway.
I should have done it right the first time, but I didn't realize how big one stack of rods is.